src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The DeleteCommand class represents a command to deleted from the task list.
+ */
+public class DeleteCommand extends Command {
+    private final int taskNumber;
```

Comment:
I've noticed several instances of the same issue; please change the variable names of constants to uppercase letters.